packages/cli/src/commands/init.ts

```diff
@@ -33,20 +43,33 @@ async function gitHubOAuth(): Promise<string> {
     return await serverResponse(httpServer);
   } catch (error) {
     // eslint-disable-next-line no-console
-    console.log(error);
+    console.error(error);
     return null;
   } finally {
     httpServer.close();
   }
+}
 
-  function serverResponse(server: HttpServer): Promise<string> {
-    return new Promise((resolve) => {
-      const interval = setInterval(() => {
-        if (server.token) {
-          clearInterval(interval);
-          resolve(server.token);
-        }
-      }, 300);
-    });
-  }
+function serverResponse(server: HttpServer): Promise<string> {
+  return new Promise((resolve) => {
+    const interval = setInterval(() => {
+      if (server.token) {
+        clearInterval(interval);
+        resolve(server.token);
+      }
+    }, 300);
+  });
+}
+
+function isOrganizationIdExist(userJwt: string): boolean {
```

Comment:
```suggestion
function isOrganizationIdExist(userJwt: string): string {
```

I believe it's a string and not a boolean :)